src/components/organisms/ProfileModal/UserInformationContent/UserInformationContent.tsx

```diff
@@ -1,179 +1,214 @@
-
-            if (!questionAnswer) return undefined;
-
-            return (
-              <div key={question.name} className="question-section">
-                <div className="question">{question.text}</div>
-                <div className="answer">{questionAnswer}</div>
-              </div>
-            );
-          })
-        : undefined,
-    [profile, profileQuestions]
+  }, [user]);
+
+  const editProfileLink = useCallback(
+    (profileLink) => () => {
+      setProfileLinkToEdit(profileLink);
+      setUserProfileMode(UserProfileMode.EDIT_PROFILE_LINK);
+    },
+    [setProfileLinkToEdit, setUserProfileMode]
   );
 
-  if (!user || !userWithId) return null;
+  if (!user) return null;
 
   return (
-    <>
-      <h1 className="title modal-title">My Profile</h1>
-
-      <div className="user-information">
-        <UserAvatar user={userWithId} showStatus large />
-        <div className="text-container">
-          <h3>{profile?.partyName || DEFAULT_PARTY_NAME}</h3>
-          <div className="ellipsis-text">{user.email}</div>
-          <UserStatusDropdown />
+    <div className="UserInformationContent">
+      <h1 className="UserInformationContent__title">My Profile</h1>
+      <div className="UserInformationContent__information">
+        <div>
+          <UserAvatar user={user} showStatus large />
         </div>
+        <div className="UserInformationContent__text-container">
+          <h3 className="UserInformationContent__user-name">
+            {user?.partyName ?? DEFAULT_PARTY_NAME}
+          </h3>
+          <div
+            title={email ?? ""}
+            className="UserInformationContent__ellipsis-text"
+          >
+            {email}
+          </div>
+          <div className="UserInformationContent__status-container">
+            <span className="UserInformationContent__status-prefix">
+              Available
+            </span>
+            <UserStatusDropdown />
+          </div>
+        </div>
+        <Button
+          customClass="UserInformationContent__edit"
+          onClick={() => setUserProfileMode(UserProfileMode.EDIT_PROFILE)}
+        >
+          <img src={editIcon} alt="edit" />
+        </Button>
       </div>
-
-      {renderedProfileQuestionAnswers}
+      {profileQuestions &&
+        profileQuestions.map((question: QuestionType) => (
+          <div key={question.name} className="question-section">
+            <div className="question">{question.text}</div>
+            <div>
+              {
+                // @ts-ignore question.name is a correct index for type User
+                (profile && profile[question.name]) ||
+                  DEFAULT_PROFILE_QUESTION_ANSWER
+              }
+            </div>
+          </div>
+        ))}
+
+      <ul className="UserInformationContent__profile-links">
+        {user?.profileLinks?.map((profileLink) => (
+          <li key={profileLink.title}>
+            {profileLink.title}{" "}
```

Comment:
```suggestion
            {profileLink.title}
```

Reply:
@mike-lvov Why are you saying to remove this here? This is what React does to ensure a space is included. Do we not want a space here?

Reply:
Looking at the wider code, this looks like it will include a space between the title and the button, which, without looking closer at the PR/designs, sounds pretty logical to me.

Reply:
> @mike-lvov Why are you saying to remove this here? This is what React does to ensure a space is included. Do we not want a space here?

I don't mind having a space there. I would expect it to be handled with CSS

Reply:
> I would expect it to be handled with CSS

Unsure in this particular instance, but generally speaking this pattern of `{" "}` is pretty standard react for when we need a space between elements in JSX. It's basically (if not literally) the equivalent of using `&nbsp;` in HTML.

Reply:
yes, that space is needed to separate the `title` and the `button` which looks like an `a`-tag (does not have any borders / margins).
![image](https://user-images.githubusercontent.com/63313732/123028783-f9e0e300-d3ad-11eb-8711-4eec9ae84f4d.png)

Reply:
as the owner of the PR I resolve this conversation leaving `{" "}` in place! 😎